mappings.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<mappings>
+  <mapping>
+    <path>~/navigation/position/latitude</path>
+    <sentence>
+      <id>RMC</id>
+      <fieldset type="coordinate">
+        <field>2</field>
+        <field>3</field>
+      </fieldset>
+      <filter type="if">
+        <field>1</field>
+        <value transform="uppercase">A</value>
+      </filter>
+    </sentence>
+  </mapping>
+
+  <mapping>
+    <path>~/navigation/position/longitude</path>
```

Comment:
I guess there are arguments for both. For now I'll keep things as they are, and won't stir up another discussion ;-)